### ISSUE TITLE
Ability to change feature color without event

### ISSUE THREAD

**Issue by RominaMartin** [CartoDB/carto-vl#1105]
I'm trying out carto vl and I wonder If there's a way to modify a geometry color obtained by carto without a map event. 
My goal is to click a button and obtain the same `Feature` object I get when I add an event like `featureClick` or so.

I'm not sure if this is related to #954

**Comment by davidmanzanares** [comment]
Hi,

With #954 the workflow will be easier.

In the meantime, you can still do it, but it will take a little bit more of code.

If I understood correctly you want something like this (pseudo-code):
```
color: feature == theFeatureImInterested ? red : green
```

This can be expressed in CARTO VL with:
```
color: blend(red, green, $featureID == @selectedID)
@selectedID: -1
```

You'll be able to change the selected feature with `viz.variables.selectedID = 1234`.

Of course, you can provide any kind of boolean-like logic as the third parameter:

```
color: blend(red, green, $featureCountry == 'Spain' or $featureCountry == 'France' )
```

Assuming you only want to have one "focused" feature this won't be a nightmare, if you plan to have more, the only way is to use `blend` in a recursive manner, this is from my expericence very difficult and I would recommend to wait for #954 if that is your case.

**Comment by RominaMartin** [comment]
Works like a charm. 

Thank you!

**Comment by davidmanzanares** [comment]
Hi,

Yes, with the trick I mentioned earlier you would need to manage that `reset` yourself by setting `selectID` to a value that is not present in your table when needed.

We merged https://github.com/CartoDB/carto-vl/pull/1085 into `master` already. However, we haven't released a version after that, and `master` is usually more unstable than our released versions.

Known problems with `master` at this moment are related to categorical properties with MVT sources and animations.

We hope to release a version with those problems fixed soon.

**Comment by davidmanzanares** [comment]
We released version `0.10.0` with #1085 included, mentioned bugs should be fixed there too.
